compiler/rustc_hir_typeck/src/coercion.rs

```diff
@@ -764,9 +764,11 @@ impl<'f, 'tcx> Coerce<'f, 'tcx> {
             self.tcx,
             self.cause.clone(),
             self.param_env,
-            ty::Binder::dummy(
-                self.tcx.at(self.cause.span).mk_trait_ref(hir::LangItem::PointerLike, [a]),
-            ),
+            ty::Binder::dummy(ty::TraitRef::from_lang_item(
```

Comment:
can you remove `ty::Binder::dummy` here?

Reply:
Yes